File: 344-ReverseString/344-ReverseString.js
// Last updated: 10/03/2026, 13:45:07
/**
 * @param {character[]} s
 * @return {void} Do not return anything, modify s in-place instead.
 */
var reverseString = function(s) {
     let left = 0;
    let right = s.length - 1;

    while (left < right) {
        [s[left], s[right]] = [s[right], s[left]];
        left++;
        right--;
    } 
};

File: 344-ReverseString/344-ReverseString.cpp
// Last updated: 09/04/2026, 09:29:21
1class Solution {
2public:
3    void reverseString(vector<char>& s) {
4        int left = 0;
5        int right = s.size() - 1;
6
7        while (left < right) {
8            swap(s[left], s[right]);
9            left++;
10            right--;
11        }
12    }
13};